### PR TITLE
feat(contract): switch to eth price getter for example

### PR DESCRIPTION
**Pull request by gskapka**
...per title. Makes for a more fun & widely applicable use-case for the likely demographic of this Truffle starter.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,14 +56,13 @@ _**`❍ truffle develop> test`**_
...
 ```
-  Contract: Diesel Price Tests
+  Contract: ETH Price Tests
     ✓ Should get contract instantiation for listening to events
     ✓ Should have logged a new Provable query
-    ✓ Callback should have logged a new diesel price (4676ms)
-    ✓ Should set diesel price correctly in contract
+    ✓ Callback should have logged a new ETH price (4676ms)
+    ✓ Should set ETH price correctly in contract
     ✓ Should revert on second query attempt due to lack of funds (120ms)
```
Comment (riccardopersiani) [inline README.md]:
```suggestion
 ✓ Should succeed on a second query attempt when sending funds (97ms)
```

File: README.md [hunk 2]
```diff
@@ -56,14 +56,13 @@ _**`❍ truffle develop> test`**_
...
 
-
   5 passing (6s)
```
Comment (riccardopersiani) [inline README.md]:
```suggestion
   6 passing (6s)
```

File: README.md [hunk 3]
```diff
@@ -56,14 +56,13 @@ _**`❍ truffle develop> test`**_
 
 
 ```
-  Contract: Diesel Price Tests
+  Contract: ETH Price Tests
     ✓ Should get contract instantiation for listening to events
     ✓ Should have logged a new Provable query
```
Comment (riccardopersiani) [inline README.md]:
```suggestion
     ✓ Should have logged a new Provable query on contract creation
```

**Review by riccardopersiani — CHANGES_REQUESTED** [review]
Tested successfully.

Small changes required.

File: README.md (hunk 1 above)
Comment (gskapka) [reply README.md]: Oops, forgot I added a new test & didn't update the `README` - thanks!

**Comment by gskapka** [comment]
Just forced this to clean up those tab issues your suggestions accidentally introduced. Should look neat again now!

File: README.md [hunk 4]
```diff
@@ -56,14 +56,13 @@ _**`❍ truffle develop> test`**_
...
-    ✓ Callback should have logged a new diesel price (4676ms)
-    ✓ Should set diesel price correctly in contract
+    ✓ Should have logged a new Provable query on contract creation
+    ✓ Callback should have logged a new ETH price (4676ms)
+    ✓ Should set ETH price correctly in contract
     ✓ Should revert on second query attempt due to lack of funds (120ms)
-
-
+    ✓ Should succeed on a second query attempt when sending funds (97ms)
   5 passing (6s)
```
Comment (riccardopersiani) [inline README.md]:
```suggestion
   6 passing (6s)
```

**Review by riccardopersiani — APPROVED** [review]
Just commit the small change in the readme

**Pull request merged**